spec/models/event_procedure_spec.rb

```diff
@@ -50,10 +50,16 @@
   end
 
   describe "validations" do
+    let(:event) { build(:event_procedure, urgency: true, procedure_attributes: { custom: true }) }
+
     it { is_expected.to validate_presence_of(:date) }
     it { is_expected.to validate_presence_of(:patient_service_number) }
     it { is_expected.to validate_presence_of(:room_type) }
     it { is_expected.to validate_presence_of(:payment) }
+
+    it "is invalid when custom and urgency are both true" do
```

Comment:
Create a context named validations and put his `it` and the `let` inside of the `validations` context